test/built-ins/Temporal/PlainDateTime/prototype/until/subseconds.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.until
+description: String argument gets cast
```

Comment:
This description doesn't seem correct for the test.

Reply:
Looks like a copy & paste relic. I've tried to improve it now.